cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
```

Comment:
We should avoid managing views and their constraints in the controller itself, it is not it's responsibility to take care of the layout, therefore we can make this controller view the `PaymentView` like this

``` swift
// inside vc

override func loadView() {
        view = PaymentView()
}

// we can also have a helper accessor to be able to work with the view properties like that
// in here we are sure that we override default view with our custom one so we can safely force unwrap
var paymentView: PaymentView { view as! PaymentView }

```